data/json/items/ammo/20x66mm.json

```diff
@@ -134,8 +133,9 @@
     "description": "20x66mm caseless shotgun rounds, solid projectile type.  Proprietary ammunition for Rivtech shotguns.  Being caseless rounds, these cannot be disassembled or reloaded.",
     "price": 1500,
     "price_postapoc": 9000,
-    "relative": { "range": 12, "pierce": 8 },
-    "proportional": { "damage": 0.65, "dispersion": 1.3 },
+    "damage": { "damage_type": "stab", "amount": 39, "armor_penetration": 8 },
```

Comment:
Is it correct that it went from `"damage": 0.65` to `"damage_type": "stab", "amount": 39`?

Reply:
Base damage, as it is derived from `"copy-from": "20x66_shot"`, is 60. `"proportional": { "damage": 0.65` means 60 x 0.65 = 39. So yeah, that's correct.

Reply:
This is because the code won't allow both `relative` and `proportional` damage objects. I'm not sure why I didn't just move this all to relative.